shop-data/src/main/java/com/maxshop/di/RepositoryModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.maxshop.di
+
+import com.maxshop.repository.CategoryRepository
+import com.maxshop.repository.CategoryRepositoryImpl
+import dagger.Binds
+import dagger.Module
+
+@Module
+internal interface RepositoryModule {
+    @Binds
+    fun getCategoryRepository(impl: CategoryRepositoryImpl): CategoryRepository
```

Comment:
would be better to use one style for all @Provides and @Binds methods. Then in future it would be easier to find them. Usually we use `provideWhat` style. For example, `provideCategoryRepository`. This style must be used for all methods in dagger modules

Reply:
fixed